src/tbb/global_control.cpp

```diff
@@ -149,17 +152,20 @@ static control_storage_instance* control_storage_instance_ptr = nullptr;
 static control_storage *controls[4] = {nullptr, nullptr, nullptr, nullptr};
 
 void global_control_acquire() {
-    control_storage_instance_ptr = new control_storage_instance{};
+    auto ptr = aligned_alloc(max_nfs_size, sizeof(control_storage_instance));
```

Comment:
MSVC does not support ``aligned_alloc``/``free``. Use ``_aligned_malloc`` and ``_aligned_free`` instead.

Reply:
I suggest to use special functions: `cache_aligned_allocate()` and `cache_aligned_deallocate()`